package io.maksymuimanov.task.exception;

public class NettyServerException extends RuntimeException {
    public NettyServerException(Throwable cause) {
        super(cause);
    }
}
